eth/downloader/downloader.go

```diff
@@ -289,6 +289,10 @@ func (d *Downloader) Synchronising() bool {
 	return atomic.LoadInt32(&d.synchronising) > 0
 }
 
+func (d *Downloader) SnapsyncInProgress() bool{
+	return d.snapSync && d.Synchronising()
```

Comment:
Hm, this probably won't work, becase `synchronizing` is only set after we find a peer to start syncing against

Reply:
Checking the `d.snapSync` should be sufficient, it's just that we currently don't set it to `0` after the sync is done